tests/integration-tests/conftest.py

```diff
@@ -512,6 +512,7 @@ def cfn_stacks_factory(request):
     # NAT Gateway not available in sae1-az2
     "sa-east-1": ["sae1-az1", "sae1-az3"],
     # m6g.xlarge instances not available in euw1-az3
+    # p4d.24xlarge instances only available in euw1-az2
     "eu-west-1": ["euw1-az1", "euw1-az2"],
```

Comment:
If p4d only exists in euw1-az2, should we remove euw1-az1?

Reply:
lol yes definitely, dumb mistake on my part.